app_radio_module/src/lora_utils.h

```diff
@@ -33,14 +33,42 @@ typedef struct __attribute__((__packed__)) {
     float temperature_tmp;
 } FAKE_SENSOR_DATA_T;
 
-
-
+/**
+ * @brief Initialize the SX1278 Radio Module
+ * @details This device is the one that uses the LoRa protocol
+ * @param dev The device to initialize
```

Comment:
Specify [in] for input variables or [out] for output variables. For example, pointers that will be modified and used after the call would be considered output.